README.md

```diff
@@ -1,70 +1,190 @@
-# Getting Started with Create React App
+TODOLIST-APP
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<a name="Richard Sikaonga"></a>
 
-## Available Scripts
+<div  align="center">
+<!--<img src="src/assets/RIYI-LOGO.png" alt="logo" width="160" border-radius=" 40px" height="auto"/>-->
+<br/>
+  <h3><b>My TodoList</b></h3>
+</div>
 
-In the project directory, you can run:
+# 📗 Table of Contents
 
-### `npm start`
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[TodoList\] ](#-todolist-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Feature ](#key-feature-)
+  - [📹 Video Presentation ](#-video-presentation-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📖 [TodoList] <a name="about-project"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+This is the web Math Calculation app that will help other developer get motivated on creating the react and API apps .
+Its Mobile vision app and can be accessed on the Mobile devices and Desktop
```

Comment:
I would like to explain that the project description should be in detail. So both technical and non-technical people can easily understand the project.